test/support/utility.ex

```diff
@@ -255,20 +255,23 @@ defmodule Membrane.VideoCompositor.Test.Support.Utils do
   end
 
   @doc """
-  Compare contents of two raw files, succeeding when `different_bytes / length < allowed_error`
+  Compare contents of two raw files, succeeding when `mse < allowed_error`
   """
   @spec compare_contents_with_error(binary(), binary(), float()) :: boolean()
-  def compare_contents_with_error(output_path, reference_path, allowed_error \\ 0.003) do
+  def compare_contents_with_error(output_path, reference_path, allowed_error \\ 1.5) do
```

Comment:
I just recalled, that since we are changing how the ref video is compared with the test output video, we should also remove platform-based separation in [`test/texture_transformations_test.exs`](https://github.com/membraneframework/membrane_video_compositor_plugin/blob/master/test/texture_transformations_test.exs#:~:text=to%20different%20references.-,%40tag%20linux%3A%20true,-test%20%221%20frame). Linux/mac separation shouldn't be required anymore.